CMakeLists.txt

```diff
@@ -248,6 +247,31 @@ configure_file(${CMAKE_SOURCE_DIR}/libraries/cli11/bash-completion/completions/c
 install(FILES libraries/cli11/bash-completion/completions/leap-util DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/bash-completion/completions COMPONENT base)
 install(FILES libraries/cli11/bash-completion/completions/cleos DESTINATION ${CMAKE_INSTALL_FULL_DATAROOTDIR}/bash-completion/completions COMPONENT base)
 
+set(TESTER_TARGETS fc secp256k1 bn256 chainbase builtins softfloat Logging IR WAST WASM Runtime eosio_chain eos-vm eosio_testlib)
```

Comment:
Is there a better way then to dump all these here like this? It seems if I'm understanding right, one option may be to instead do something like
```
install(TARGETS fc EXPORT dev ...)
```
and then just
```
export(EXPORT dev ...)
```

Reply:
For cmake 3.13+, `install(TARGETS fc secp256k1 ... EXPORT dev ...)` and `then export(EXPORT dev ...)` would work. However, it won't work for my hack for cmake 3.8. Another point is that listing `fc` is not enough, cmake requires the complete list of the libraries explicitly. However, if you missing any of the required libraries, cmake would generate errors and tell you which library is missing during configuration.

Reply:
I'm not saying to use `install(TARGETS fc secp256k1 ... EXPORT dev ...)` but rather add `EXPORT dev` to the existing install declaration that's already present for each target, for example here
https://github.com/AntelopeIO/leap/blob/2640009e79bb7eca2195e5ff187f19c514edd219/libraries/libfc/CMakeLists.txt#L144-L147
and then in the root CMakeLists.txt it just becomes `export(EXPORT dev` which gobbles up everything in the `dev` "export group".

Reply:
I reread what you originally wrote... yeah if doing that the clear benefit is not needing `COMPONENT` & `EXPORT` for each target's `install()` (which can be high effort to add for targets in submodules etc)

Reply:
I tried what you have said to have EXPORT in each individual target. It does have higher maintenance cost as you said. However, it would work with cmake 3.8.0 out of box without my hack.